uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss

```diff
@@ -7,6 +7,14 @@
 
 // Customizations specific for T&A
 .civihr-ui-select {
+  * {
+    border-color: $brand-primary !important;
+  }
+
+  .select2-search-choice {
```

Comment:
why is this necessary? where did that 11px font size come from?

Reply:
I already mentioned it in the documentation, please check.

> it overrides the css from sites/all/modules/civicrm/css/civicrm.css

```
.crm-container div.select2-container {
    font-size: 11px;
}
```